src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -142,24 +165,24 @@ public async Task InitAsync()
                 }
                 catch (TimeoutException)
                 {
-                    if (timeout == TimeSpan.Zero && _logger.IsInfoEnabled)
+                    if (timeout == TimeSpan.FromMilliseconds(16) && _logger.IsInfoEnabled)
```

Comment:
Why exactly 16 milliseconds? Is it correct?

Reply:
Yes, it's timout's initial value. There is no scenario where it can be TimeSpan.Zero

Reply:
Let's make this a constant value, instead.

Reply:
ok